apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestInputActivity.java

```diff
@@ -141,7 +141,7 @@ public void stopAudio() {
 
     @Override
     protected void toastPauseError(int result) {
-        showToast("Pause not implemented. Returned " + result);
+        showToast("Pause not implemented. Returned " + StreamConfiguration.convertErrorToText(result));
```

Comment:
Can just say "Pause failed with -890, ErrorUnimplemented" or whatever the error text is.